microservices-orders/scripts/run-services.sh

```diff
@@ -5,7 +5,8 @@ set -m
 BOOTSTRAP_SERVERS=${BOOTSTRAP_SERVERS:-broker:9092}
 SCHEMA_REGISTRY_URL=${SCHEMA_REGISTRY_URL:-http://schema-registry:8081}
 RESTPORT=${RESTPORT:-18894}
-JAR=${JAR:-"/usr/share/java/kafka-streams-examples/kafka-streams-examples-$CONFLUENT-standalone.jar"}
+JAR_DEFAULT="/usr/share/java/kafka-streams-examples/"$(ls /usr/share/java/kafka-streams-examples/ | grep standalone | awk '{print $NF;}')
```

Comment:
I don't think we need to be specific about the actual jar file name, we can use wildcards:

JAR=${JAR:-"/usr/share/java/kafka-streams-examples/kafka-streams-examples-*.jar"}

But for the local build part, I think we should add a `mvn clean` stage in the `get_and_compile_kafka_streams_examples` function so that it will clean out any old build artifacts before running.  Otherwise, the wildcard might be an issue with previous artifacts.